doc/source/serve/deploy_model_tutorial.rst

```diff
@@ -0,0 +1,315 @@
+    ...
+    return preds[0]
+
+Ray Serve needs to run on top of a Ray Cluster, so we start a local one:
+
+.. code-block:: python
+
+  ray.init()
+
+.. note::
+
+  ``ray.init()`` will start a single-node Ray cluster on your local machine, 
+  which will allow you to use all your CPU cores to serve requests in parallel. 
+  To start a multi-node cluster, see :doc:`../cluster/index`.
+
+Next, we start the Ray Serve runtime:
+
+.. code-block:: python
+
+  serve.start()
+
+.. warning::
+
+  When the Python script exits, Ray Serve will shut down.  
+  If you would rather keep Ray Serve running in the background you can use 
+  ``serve.start(detached=True)`` (see :doc:`deployment` for details).
+
+Now that we have defined our ``summarize`` function, launched a Ray Cluster, 
+and started the Ray Serve runtime, we can define a function to accept HTTP 
+requests and route them to the ``summarize`` function. Since ``summarize`` 
+takes in article ``text``, we need to take that in as a parameter in the HTTP 
+request URL. We can define a function called ``router`` that takes in a Starlette 
+``request`` object. ``router`` then looks for the ``txt`` parameter in the 
+Starlette ``request`` object to find the requested article text that must be summarized. 
+It then passes in the article text, as well its length, into the 
+``summarize`` function and returns the value. We also add the decorator 
+``@serve.deployment`` to the ``router`` function to turn it into a Serve 
+``Deployment`` object.
+
+.. code-block:: python
+
+  @serve.deployment
+  def router(request):
+    txt = request.query_params["txt"]
+    return summarize(txt, max_length=len(txt))
+
+.. tip::
+  This routing function's name doesn't have to be ``router``. 
+  It can be any function name as long as the corresponding name is present in 
+  the HTTP request. If you want the function name to be different than the name 
+  in the HTTP request, you can add the ``name`` keyword parameter to the ``@serve.deployment`` 
+  decorator to define the name sent in the HTTP request. For example, if the 
+  decorator is ``@serve.deployment(name="responder")`` and the key signature is
+  ``def request_manager(request)``, the HTTP request should use ``responder``, 
+  not ``request_manager``. If no ``name`` is passed into ``@serve.deployment``, 
+  the ``request`` will by default use the function's name instead. 
+  For example, if the decorator is ``@serve.deployment`` and the function's key 
+  signature is ``def manager(request)``, the HTTP request should use ``manager``.
+
+Since ``@serve.deployment`` makes ``router`` a ``Deployment`` object, it can be deployed using ``router.deploy()``:
+
+.. code-block:: python
+
+  router.deploy()
+
+Once we deploy ``router``, we can query the model over HTTP. 
+And with that, we can run our model on Ray Serve! 
+Here's the full Ray Serve deployment script that we built for our model:
```

Comment:
We probably want to add 1-2 sentences on how to run this (i.e. To deploy, first start ray by running ray start --head, then run your python script: python example.py)